### DISCUSSION TITLE
Organizer and attendee issue

### DISCUSSION THREAD

**Discussion by joseserrano15** [spatie/icalendar-generator#62] [Q&A]
Hi, I have a problem with the organizer and attendee function, I´m using it in my code but when I generate my calendar file and import it into google calendar, the calendar don´t show the information about attendee or organizer. Anyone know what could be the reazon? 
Thanks!

**Comment by rubenvanassche** [comment]
Could you show me an example of such calendar?

**Reply by josephdicdican** [reply]
Hi, I'm using icalendar-generator as well as of  Feb 16, 2022 to generate ics file and having similar issue on attendees.

I use this code to generate a sample ics to test it out:
![image](https://user-images.githubusercontent.com/15903512/154187536-4e4fad71-e4d1-4a2c-8179-35ccf5586cb3.png)

I then manually import it to google & open via outlook calendar respectively.

Here is the google event with no attendees:
![image](https://user-images.githubusercontent.com/15903512/154187452-56cbdd62-7ca7-4b7d-8ce0-713e54650c36.png)

For outlook, no attendees as well:
![image](https://user-images.githubusercontent.com/15903512/154187572-62b334e2-25aa-49b8-aa25-2ae014d9bb69.png)

Am I missing anything? I followed the readme examples in the repo but couldn't get it to work. Let us know your thoughts, thanks.

**Reply by rubenvanassche** [reply]
I don't think it is possible (https://support.google.com/calendar/thread/20792476/import-multiple-calendar-vevent-in-one-ics-with-gmail-ics-attachment?hl=en)

When I import a calendar within Apple iCalendar the attendees show up just fine. I guess Microsoft and Google have disabled this 😢
